app/src/main/java/com/myvetpath/myvetpath/SubDetailsActivity.java

```diff
@@ -53,6 +60,34 @@ protected void onCreate(Bundle savedInstanceState) {
         }
         TextView internalIDText = findViewById(R.id.subInternalID);
         internalIDText.setText(Integer.toString(internalId));
+
+        //set images
```

Comment:
Great job here! Images look good on the screen! Later we might need to make sure images are oriented correctly or allow the user to open them in the gallery to zoom and save them. But that can be done later.

Reply:
Yup I agree. There are definitely some ways to enhance the user experience regarding pictures.